tests/parm/rrfs_conus13km_hrrr.nml.IN

```diff
@@ -198,6 +198,7 @@
     iopt_snf  = 4
     iopt_tbot = 2
     iopt_stc  = 3
+    iopt_trs  = 2
```

Comment:
This namelist template is for FV3_HRRR suite, which uses RUC lsm. Why it needs those iopt_* options for NoahMP lsm?

Reply:
They were there from the beginning. So I made the change for consistence.

Reply:
Ok, indeed some RTs don't need those parameters. But I will keep them.